client/db.go

```diff
@@ -196,6 +196,14 @@ type Store interface {
 		transform immutable.Option[model.Lens],
 	) ([]CollectionDefinition, error)
 
+	// RefreshViews refreshes the caches of all views matching the given options.  If no options are set all views
+	// will be refreshed.
+	//
+	// The cached result is dependent on the ACP settings of the source data and the permissions of the user making
```

Comment:
thought: This makes me think that there might be need for setting who is allowed to cache views on a given node. Otherwise this could be abused.

Reply:
Yes of course, we'll either want it covered by admin ACP or allow users to own views.

Reply:
Might be worth documenting this in: https://github.com/sourcenetwork/defradb/issues/2640